clients/imodels-client-management/src/operations/imodel/IModelOperations.ts

```diff
@@ -235,14 +244,21 @@ export class IModelOperations<TOptions extends OperationOptions> extends Operati
     return result;
   }
 
-  protected getCreateEmptyIModelRequestBody(iModelProperties: IModelProperties): object {
-    return {
+  protected getCreateEmptyIModelRequestBody(iModelProperties: IModelProperties): IModelProperties {
+    const result: IModelProperties = {
       iTwinId: iModelProperties.iTwinId,
       name: iModelProperties.name,
       description: iModelProperties.description,
       extent: iModelProperties.extent,
-      containersEnabled: iModelProperties.containersEnabled
+      containersEnabled: iModelProperties.containersEnabled,
+      creationMode: iModelProperties.creationMode
     };
+
+    if (iModelProperties.geographicCoordinateSystem && iModelProperties.creationMode === "empty") {
```

Comment:
I see this getCreateEmptyIModelRequestBody method is also used in createFromTemplate code.
So it seems these GCS changes won't work with iModel creation from template. 
Probably we could move validation to createEmpty method (throw if creation mode not specified and gcs is specified).

Reply:
Makes sense - thanks and done.